ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -4383,7 +4385,6 @@ presubmits:
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-installer-master-images
     rerun_command: /test images
-    skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)
```

Comment:
Why are we removing this? It is related to the change to `always_run` but I don't understand the motivation behind this change and its relation to the move to WWT environment for azurestack.